### DISCUSSION TITLE
How can I display only the line and not the In/out boxes on my frame

### DISCUSSION THREAD

**Discussion by xwhiz** [roboflow/supervision#197] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

I want to show only the line and process the in/out in the background. How can I possibly do it?

**Comment by SkalskiP** [comment]
Hi @hamza-student 👋🏻 Let me move this question to our QA section in discussions first.

**Reply by xwhiz** [reply]
Thanks. Kindly guide me.

**Comment by SkalskiP** [comment]
@hamza-student you intend to just draw the line? Don't want to perform any calculations?

**Reply by xwhiz** [reply]
thank you for responding, I want to draw the line and have the functionality, but the in out boxes should not be on top of the line and would be somewhere on side of the screen.

**Comment by maddust** [comment]
@CodeWhizHamza you can use your start and end points to draw a  line by using draw_line util 
then then simply do not annotate the LineZone counter .to your frame
